setup.py

```diff
@@ -88,6 +88,7 @@ def recursively_include(results, directory, patterns):
       description='Android APK packager for Python scripts and apps',
       long_description=long_description,
       long_description_content_type='text/markdown',
+      python_requires="==3.*,>=3.6.0",
```

Comment:
Nice! Just curious isn't this redundant? Don't we only need `python_requires="==3.*,>=3.6.0"`.
If you added it in case of Python 4, I would say keep it simple